### PR TITLE
[DONE] Remove BBB module and add sipmediagw feature

### PR DESCRIPTION
**Pull request by LoicBonavent**
* Removal of the old BBB module (replaced by the meetings module)
* Delete MEETING_WEBINAR_SIPMEDIAGW_URL and MEETING_WEBINAR_SIPMEDIAGW_TOKEN parameters management (replaced by live gateway feature)
* Add sipmediagw_server_url and sipmediagw_server_password to live gateway class (meeting/models.py)
* Manage request URLs from SIPMediaGW server (hashkey with a / before the last 10 characters)

### CODE REVIEW

File: pod/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -11238,6 +10829,337 @@ msgstr "Résultats de la recherche"
 msgid "Esup-Pod xAPI"
 msgstr "xAPI Esup-Pod"
 
+#~ msgid ""
```
Comment (Badatos) [inline pod/locale/fr/LC_MESSAGES/django.po:10832]: Peux-tu supprimer toute cette partie en commentaires stp ?

**Review by Badatos — APPROVED** [review]
Merci Loic,
Voila un sacré ménage de fait ;)

File: pod/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (LoicBonavent) [reply pod/locale/fr/LC_MESSAGES/django.po]: Yes, fait !

**Review by fanfounet — APPROVED** [review]
Code très propre, bien documenté !

**Pull request merged**